README.md

```diff
@@ -11,6 +11,7 @@ It runs in the same process as the application, so communication overhead is min
 - [Serilog](#serilog)
 - [Microsoft.Extensions.Logging](#microsoftextensionslogging)
 - [ETW (Event Tracing for Windows)](#etw-event-tracing-for-windows)
+- [Application Insights](#application-insights-input)
```

Comment:
I am a bit hesitant to emphasize AI + EventFlow phase 1--I think we will have much better story after phase 2 and esp. phase 3. At that time I will put a big, red text at the beginning of the README saying "we have this new AI SDK and you are going to love it".